operator-framework-core/src/main/java/io/javaoperatorsdk/operator/Operator.java

```diff
@@ -253,13 +258,21 @@ private void throwMissingCRDException(String crdName, String specVersion, String
     return false;
   }
 
-  private static class ControllerRef {
+  public static class ControllerRef {
     public final ResourceController controller;
     public final ControllerConfiguration configuration;
 
     public ControllerRef(ResourceController controller, ControllerConfiguration configuration) {
       this.controller = controller;
       this.configuration = configuration;
     }
+
+    public ResourceController getController() {
```

Comment:
Why add getters if the instance variables are public final… Use one or the other :)

Reply:
I think this is a leftover when using the `ResourceController` as part of some stream operation so getter/setter where require for method references